src/aro/Parser.zig

```diff
@@ -7226,8 +7241,7 @@ fn unExpr(p: *Parser) Error!Result {
                 const lhs_ty = p.nodes.items(.ty)[@intFromEnum(data.member.lhs)];
                 if (lhs_ty.hasAttribute(.@"packed")) {
                     const record = lhs_ty.getRecord().?;
-                    const mapper = p.comp.string_interner.getSlowTypeMapper();
-                    const extra = Diagnostics.Message.Extra{ .record_member = .{ .record = mapper.lookup(record.name), .member = mapper.lookup(record.fields[data.member.index].name) } };
+                    const extra = Diagnostics.Message.Extra{ .str = try p.packedMemberAccessStr(record.name, record.fields[data.member.index].name) };
                     try p.errExtra(.packed_member_address, orig_tok_i, extra);
```

Comment:
You can use the `errStr` function here with the string

Reply:
+